Readme.md

```diff
@@ -79,7 +79,7 @@ This screenshot shows an Airbrake error send from this module:
 
 ## Features
 
-* Automatically add `process.env` as well as other information when sending notifications
```

Comment:
"as well as other information" was a feature. Specifically what is the other information, is it useful to list this as a feature?